### PR TITLE
Add jest-watch-typeahead

### PR DESCRIPTION
**Pull request by RobertBroersma**
This adds some cool features to Jest watching: https://github.com/jest-community/jest-watch-typeahead

Jest Watch Typeahead

After this PR you should be able to search and isolate tests when running `yarn rw test` accross all sides!

### CODE REVIEW

**Comment by peterp** [comment]
Oooh, this is freaking awesome! <3

**Comment by peterp** [comment]
Yo, @RobertBroersma is this ready to merge/ review?

**Comment by RobertBroersma** [comment]
@peterp Yes it is! I had some weird stuff going on with the new dependency on Jest Typeahead, because they are not explicitly imported anywhere I think? So it would be good to check if they are properly pulled in when this is merged, but I'm not sure how?

**Comment by peterp** [comment]
@RobertBroersma Hard to follow why it wasn't working, but let's merge this... if it's working then things should be good?

**Review by peterp — APPROVED** [review]
